config/schedule.rb

```diff
@@ -0,0 +1,3 @@
+every 1.day do
+  runner "Card.mail_cards_to_review", output: 'log/cron.log'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.